src/App.js

```diff
@@ -1,16 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  //state
+  const [messageData, setMessageData] = useState(chatMessages);
+  let totalLikes = 0;
+
+  const updateMessageData = (updatedMessage) => {
+    const messages = messageData.map((message) => {
+      if (message.id === updatedMessage.id) {
+        return updatedMessage;
+      } else {
+        return message;
+      }
+    });
+    setMessageData(messages);
+  };
```

Comment:
I might consider refactoring this, so that it doesn't update ALL of the message data, just the `liked` property